recipes/pycsep/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "pycsep" %}
+{% set version = "0.1.0.dev0" %}
```

Comment:
Is there a non `dev0` version of the package available?

Reply:
Right now there isn't. Is this required for a feedstock?

Reply:
@conda-forge/staged-recipes i bumped the version to 0.1.0 and updated the meta.yaml accordingly